docs/UserGuide.md

```diff
@@ -407,14 +409,14 @@ Removes all contacts in the app.
 </div>
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-Please be mindful that entering this command will wipe out all your contacts stored in SociaLite!
+**Please be mindful that entering this command will wipe out **all your contacts** stored in SociaLite! This cannot be undone!**
```

Comment:
I haven't checked the exact rendering of this on the github page since it is not deployed, but this might bold everything but "all your contacts" here. Not very sure!